Casks/roland-quad-capture-usb-driver.rb

```diff
@@ -72,7 +72,7 @@
   ]
 
   caveats do
-    license "https://www.roland.com/us/support/by_product/quad-capture/updates_drivers/#{version.after_colon}"
+    license "https://www.roland.com/us/support/by_product/quad-capture/updates_drivers/#{version.csv.third}"
```

Comment:
```suggestion
    license "https://www.roland.com/us/support/by_product/quad-capture/updates_drivers"
```
`version.csv.third` is `nil`.

Reply:
It's actually the 2nd part of the comma split value

Reply:
Thanks @carlocab @SMillerDev!